se_req_mve/models/experiment_flow/flow.dag.yaml

```diff
@@ -50,6 +50,7 @@ node_variants:
             path: prompts/hypothesis001.jinja2
           inputs:
             deployment_name: gpt-35-turbo
+            max_tokens: 20
```

Comment:
@dlambright03  what value have you found to be effective for max_token?

Reply:
I’ve been using 20 because it’s safely large enough for our expected result without being way too big